composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "mantisbt/mantisbt",
+    "description": "Mantis Bug Tracker",
+    "type": "project",
+    "require": {
+        "slim/slim": "^3.0"
+    },
+    "license": "GPL v2",
+    "authors": [
+        {
+            "name": "Victor Boctor",
+            "email": "vboctor@gmail.com"
+        }
+    ]
```

Comment:
Depending on what the future plans for use of Composer are, additional information may be useful (e.g. keywords, homepage, support, PHP version and extensions requirements), etc.